include/GcInfo/Target.h

```diff
@@ -17,10 +17,19 @@
 #define GCINFO_TARGET_H
 
 #include "global.h"
+#include "corinfo.h"
 
 #if (defined(_TARGET_X86_) || defined(_TARGET_X64_) || defined(_TARGET_AMD64_))
 
-// Define DWARF encodings for registers
+// Identify the frame-pointer register number
```

Comment:
@russellhadley: This is the main change in the update -- use register number definitions from ICorDebugInfo.
The original definitions in Target.h were copied from RYU Jit. 
I checked that those register enumerations are the same as the ones in ICorDebugInfo.